### PR TITLE
Attempt fix SLE build on OBS

### PR DESCRIPTION
**Pull request by Waqar144**
Maybe `aspell-devel` would work there. Not sure

### CODE REVIEW

File: obs/qownnotes.spec [hunk 1]
```diff
@@ -24,6 +24,13 @@ BuildRequires:  qt5-qtwebsockets-devel
...
 Requires:       qt5-qtsvg qt5-qtxmlpatterns qt5-qtwebsockets
+%endif
+
+%if 0%{?sle_version} == 150000 || 0%{?sle_version} == 150100 && !0%{?is_backports}
+BuildRequires:  libqt5-qtbase-devel libqt5-qtdeclarative-devel libQt5Svg-devel libQt5XmlPatterns-devel libqt5-qtwebsockets-devel
+#BuildRequires:  aspell-devel
+BuildRequires:  update-desktop-files
+Requires:       libQt5Svg5 libQt5Sql5 libQt5Sql5-sqlite libQt5Gui5 libQt5Network5 libQt5Widgets5 libQt5Xml5 libQt5XmlPatterns5 libQt5PrintSupport5 libQt5WebSockets5
 
 %else
```
Comment (pbek) [inline obs/qownnotes.spec]: This else-clause will now also be execute for all other distributions other then SLE, don't you think? I'll try to put your new if-statement inside the old else-clause.

Comment (pbek) [reply obs/qownnotes.spec]: so just

```
%if 0%{?sle_version} == 150000 || 0%{?sle_version} == 150100 && !0%{?is_backports}
#BuildRequires:  aspell-devel
%else
BuildRequires:  libaspell15
%endif
```

Comment (Waqar144) [reply obs/qownnotes.spec]: maybe it's aspell-devel for both. I think libaspell15 is something different.. not really sure though

Comment (pbek) [reply obs/qownnotes.spec]: we'll see, I manually added my lines in OBS, currently building

Comment (pbek) [reply obs/qownnotes.spec]: SLE 15 is already built

Comment (pbek) [reply obs/qownnotes.spec]: Looking good. Thanks for the if-statement!

Comment (Waqar144) [reply obs/qownnotes.spec]: Great :tada:

**Pull request closed without merging**